### DISCUSSION TITLE
Sobre las notas de la I2

### DISCUSSION THREAD

**Discussion by Fernanda-Bley** [IIC2133-PUC/2024-1#197] [Ayuda] [answered]
Les escribo porque ayer se cumplieron los 15 días habiles y aun no recibimos las notas ¿Cuando estaran disponibles?

**Comment by benjavicente** [comment, marked as answer]
Los días inhábiles son sábados, domingos o festivos. La prueba fue el martes 18 de junio a las 17:30.
- [+1] miércoles 19 de junio
- Jueves 20 de junio: Día Nacional de los Pueblos Indígenas. Feriado Legal
- [+1] viernes 21 de junio
- [+5] semana de 24 a 28 de junio
- [+5] semana de 1 a 5 de julio
- [+3] de lunes 8 a miércoles 10 de julio

Por lo que con 15 días hábiles, queda para el 10 de julio y no hoy 6 como dices.

---

Las notas las queremos tener muy pronto, antes de la I3.

Pero de todas formas, estamos dentro del plazo establecido en el Reglamento del Estudiante de Pregrado de la UC.
